kvclient_ops.go

```diff
@@ -71,7 +71,9 @@ func kvClient_SimpleCall[Encoder any, ReqT any, RespT any](
 		return res.Result.(RespT), res.Err
 	case <-ctx.Done():
 		if !pendingOp.Cancel() {
-			<-resulter.Ch
+			res := <-resulter.Ch
```

Comment:
Not certain about this but seems like we should return the actual result if we have it. Unsure about expectations as a user here though.